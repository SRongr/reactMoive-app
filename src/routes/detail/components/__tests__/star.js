import React, { Component } from 'react';
import Star from '../star'
import renderer from 'react-test-renderer'

it('渲染正常',()=>{
const tree = renderer.create(<Star value = {7}/>).toJSON();
    expect(tree).toMatchSnapshot();

})