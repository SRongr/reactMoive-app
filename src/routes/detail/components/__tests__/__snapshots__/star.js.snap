// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`渲染正常 1`] = `
<div
  className="starWrapper"
  style={
    Object {
      "bcakgroundSize": "16px",
      "height": "16px",
      "width": "80px",
    }
  }
>
  <div
    className="starContent"
    style={
      Object {
        "bcakgroundSize": "16px",
        "width": "NaNpx",
      }
    }
  />
</div>
`;
